content/en/developers/write_agent_check.md

```diff
@@ -40,11 +40,11 @@ The check itself inherits from `AgentCheck` and sends a gauge of `1` for `hello.
 {{< code-block lang="python" filename="hello.py" >}}
 # the following try/except block will make the custom check compatible with any Agent version
 try:
-    # first, try to import the base class from old versions of the Agent...
-    from checks import AgentCheck
+    # first, try to import the base class from new versions of the Agent...
+    from datadog_checks.base.checks import AgentCheck
```

Comment:
The path used for generating new integrations is `from datadog_checks.base import AgentCheck`, I think we should use that one instead.

@ofek Could you confirm?

https://github.com/DataDog/integrations-core/blob/28ee8865471050ec6b0e9423b3340683caff504e/datadog_checks_dev/datadog_checks/dev/tooling/templates/check/%7Bcheck_name%7D/datadog_checks/%7Bcheck_name%7D/%7Bcheck_name%7D.py#L1

Reply:
Yes, we only show the verbose way in the context of migration https://docs.datadoghq.com/agent/guide/python-3/#package-imports